buildscripts/build_all_pkgs.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+compile_hologram.sh --deps || exit 1
+build_deb_pkgs.sh --no-compile || exit 1
+build_osx_pkgs.sh --no-compile || exit 1
```

Comment:
There are not that many lines here, but maybe it's worth to increment exit codes so we end up with exit 1, exit 2 and so on?